app/serializers/participant_declaration_serializer.rb

```diff
@@ -18,5 +18,9 @@ class ParticipantDeclarationSerializer
     declaration.declaration_date.rfc3339
   end
 
+  attribute :voided do |declaration|
+    declaration.voided || false
+  end
+
```

Comment:
Are we doing this as opposed to having it a declaration state? Do you see this changing as a change to the API later?

Reply:
I think it can be greatly simplified now - this code was written on top of a previous version which did indeed store voided data on profile declarations.

I think now it should be just `attribute :voided`. Good spot!